lib/Redis.ts

```diff
@@ -537,6 +542,23 @@ class Redis extends Commander implements DataHandledable {
     return command.promise;
   }
 
+  private setSocketTimeout() {
+    this.socketTimeoutTimer = setTimeout(() => {
+      this.stream.destroy(new Error(`Socket timeout. Expecting data, but didn't receive any in ${this.options.socketTimeout}ms.`));
+      this.socketTimeoutTimer = undefined;
+    }, this.options.socketTimeout);
+
+    // this handler must run after the "data" handler in "DataHandler"
+    // so that `this.commandQueue.length` will be updated
+    this.stream.once("data", () => {
+      console.log('GOT DATA, CLEARING TIMER');
```

Comment:
This console.log most likely doesn't belong here 😉

Reply:
And I thought everyone gonna love this new random console.log.. silly me.. ;)
Fixed in #1883, version 5.4.1 is on npm.. :)
Sorry about it..